searchcore/src/vespa/searchcore/config/proton.def

```diff
@@ -483,6 +483,11 @@ hwinfo.cpu.cores int default = 0 restart
 ## See shared_threading_service_config.cpp for details on how the thread pool sizes are calculated.
 feeding.concurrency double default = 0.2 restart
 
+## A number between 0.0 and 1.0 telling how nice the feed and background threads shall be.
+## A value of 0.0, which is default, means 'not any nicer than anyone else'.
+## The scale from 0.0 to 1.0 is not linear. It is OS specific. A large number means 'not less nice than a lower number'.
```

Comment:
This comment makes the feature not harder to misunderstand

Reply:
Removed the last sentence.